packages/react-components/src/molecules/MembersList.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```

Comment:
This is extracted from `TeamMembersSection`, but then enhanced by functionality so may deserve full re-review